packages/react/src/hooks/public/use-remote-hosts.ts

```diff
@@ -0,0 +1,48 @@
+import { PublicClient } from '@abstract-money/core'
+import { UseQueryOptions, useQuery } from '@tanstack/react-query'
```

Comment:
should use `UseQueryParameters` instead

i.e.
`import { UseQueryParameters, useQuery } from '../types/queries'`

Reply:
not critical though